src/test/java/org/candlepin/subscriptions/marketplace/MarketplaceSubscriptionIdProviderTest.java

```diff
@@ -54,7 +56,7 @@
 
 @SpringBootTest
 @ExtendWith(MockitoExtension.class)
-@ActiveProfiles({"marketplace", "test"})
+@ActiveProfiles({"marketplace", "worker", "test"})
 class MarketplaceSubscriptionIdProviderTest {
```

Comment:
Changes in this file unintentionally included?

Reply:
No, without the worker profile, this complains about missing dependencies for injection since the rest of this code was added.